SignalUI/AttachmentApproval/AttachmentPrepViewController.swift

```diff
@@ -219,12 +219,15 @@ public class AttachmentPrepViewController: OWSViewController {
     private func presentFullScreen(viewController: UIViewController) {
         if let presentedViewController = presentedViewController {
             owsAssertDebug(false, "Already has presented view controller. [\(presentedViewController)]")
-            presentedViewController.dismiss(animated: false)
+            presentedViewController.dismiss(animated: false) { [weak self] in
+                self?.presentFullScreen(viewController: viewController)
+            }
+            return
         }
 
         viewController.modalPresentationStyle = .fullScreen
         zoomOut(animated: true) { [weak self] in
-            self?.presentFullScreen(viewController, animated: false)
+            self?.present(viewController, animated: false, completion: nil)
```

Comment:
Especially since this isn't related to the fix, I'm more inclined to drop the `viewController.modalPresentationStyle = .fullScreen` here, and continue to call `self.presentFullScreen` here.